lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -237,8 +251,46 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
     }
 
     func textFieldDidEndEditing(_ textField: UITextField) {
+        activeField = nil
         guard let textFieldSubLayers = textField.layer.sublayers else { return }
         let bottomBorder = textFieldSubLayers[0] as CALayer
         bottomBorder.backgroundColor = LucienTheme.silver.cgColor
     }
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if let nextTextField = textField.superview?.superview?.viewWithTag(textField.tag + 1) as? UITextField {
+            nextTextField.becomeFirstResponder()
+        } else {
+            textField.resignFirstResponder()
+        }
+        return false
+    }
+
+    // MARK: - Keyboard Methods
+
+    func registerForKeyboardNotifications() {
+        NotificationCenter.default.addObserver(self, selector: #selector(AddComicViewController.keyboardWasShown), name: NSNotification.Name.UIKeyboardWillShow, object: nil)
+    }
+
+    func deregisterFromKeyboardNotifications() {
+        NotificationCenter.default.removeObserver(self, name: NSNotification.Name.UIKeyboardWillShow, object: nil)
+    }
+
+    @objc func keyboardWasShown(notification: NSNotification) {
+        guard let info = notification.userInfo,
+              let keyboardSize = (info[UIKeyboardFrameBeginUserInfoKey] as? NSValue)?.cgRectValue.size else {
+                return
```

Comment:
Formatting on this is a little off. Check the swift style guide for how to format multi-assignment guards. https://github.com/IntrepidPursuits/swift-style-guide#guard-statements